modules/dreamview/frontend/src/styles/main.scss

```diff
@@ -661,6 +661,94 @@ body {
         }
     }
 
+    .show-electricity{
```

Comment:
1. extra space between the name and "{".
2. please choose a noun to name the class (e.g. electricity-status)

spot format issues elsewhere as well, please also use formatter to fix css format.